libs/geoservices/covid/src/lib/pages/maps/components/time-map/time-map.component.html

```diff
@@ -0,0 +1,50 @@
+<section>
```

Comment:
Get rid of this `section`. It adds a margin to the top and bottom of the map and it's not particularly meaningful in this page.

Reply:
removed it